cpp/test/cpu_operations_test/Inverse_test.cc

```diff
@@ -47,7 +47,7 @@ class InverseTest : public ::testing::Test {
 };
 
 
-typedef ::testing::Types<int, float, double> MyTypes;
+typedef ::testing::Types</*int,*/float, double> MyTypes;
```

Comment:
/* and */ are not recommended in google code style, could you just take out int?